.github/workflows/e2e-conformance-trigger.yaml

```diff
@@ -0,0 +1,19 @@
+name: E2EConformanceTrigger
```

Comment:
possible to organize this e.g. /workflows/periodics/conformance?

Reply:
It isn't unfortunately. GH enforces that all workflows live in the `workflows` directory: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

Reply:
I wish they didn't do that. It would be nice to enforce more organizational structure to the workflows section